kong/plugins/ai-request-transformer/handler.lua

```diff
@@ -0,0 +1,74 @@
+local _M = {}
+
+-- imports
+local kong_meta     = require "kong.meta"
+local fmt           = string.format
+local llm           = require("kong.llm")
+--
+
+_M.PRIORITY = 777
+_M.VERSION = kong_meta.version
+
+local function bad_request(msg)
+  kong.log.info(msg)
+  return kong.response.exit(400, { error = { message = msg } })
+end
+
+local function internal_server_error(msg)
+  kong.log.err(msg)
+  return kong.response.exit(500, { error = { message = msg } })
+end
+
+local function create_http_opts(conf)
+  local http_opts = {}
+
+  if conf.http_proxy_host then -- port WILL be set via schema constraint
```

Comment:
same comment as ai-response-transformer re: schema constraint for host/port

Reply:
Fixed them both.